cliquenet/src/lib.rs

```diff
@@ -164,6 +172,19 @@ struct Server<T: tcp::Listener> {
     ping_interval: Interval,
 }
 
+/// A connect task.
+#[derive(Debug)]
+struct ConnectTask {
+    h: AbortHandle,
```

Comment:
Perhaps a slightly longer name? :)

Reply:
It mirros `IoTask` which has `rh` and `wh` (because it stores 2 `AbortHandle`s).